Task-5/server.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"httpserver/dbconnection"
+	"httpserver/handler"
+	"log"
+	"net/http"
+)
+
+func main() {
+
+	// This will connect to the database and the instance of Database is exported
+	dbconnection.ConnectDatabase()
```

Comment:
@Vatsal-S-Patel 

where is error? every function should return and handle error.